docker/runit/service/spark/run

```diff
@@ -31,6 +31,14 @@ function set_log_level() {
         ${SPARK_DIST_PATH}/conf/log4j.properties.template >${SPARK_DIST_PATH}/conf/log4j.properties
 }
 
+function set_statsd_address() {
```

Comment:
NIT: This is a duplication of the templating in `spark-env.h`.

Reply:
Good catch. We could just delete?

Reply:
I don't know if we can just delete it. Are there cases where this code is executed and not `spark-env.h`?